src/main/java/seedu/duke/UI.java

```diff
@@ -0,0 +1,170 @@
+package seedu.duke;
+
+import java.util.ArrayList;
+import java.util.NoSuchElementException;
+import java.util.Scanner;
+
+
+public class UI {
+
+    public static String separationLine = "----------------------------------------";
+
+    public static void printSeparationLine() {
+        System.out.println(separationLine);
+    }
+
+    /*
+    Function to get input from Command Line Interface
+     */
+    public static String getInput() {
+        Scanner in = new Scanner(System.in);
+        String input = "";
+        try {
+            input = in.nextLine();
+        } catch (Exception e) {
+            System.out.println(e.getMessage());
+        }
+        return input;
+    }
+
+    /**
+     *Message to be printed in the beginning;
+     */
+    public static void helloMessage() {
+        printSeparationLine();
+        System.out.println("Welcome to PlanIT!");
+        System.out.println("Start planning out your 4 years in NUS with us!");
+        printSeparationLine();
+    }
+    /**
+     *Message to be printed in the end
+     */
+    public static void endMessage() {
+        System.out.println("Thank you for using PlanIT!");
+        System.out.println("See you again next time!");
+    }
+    /**
+     * Message to be printed after adding a module
+     */
+    public static void addMessage(String modCode, String semester, String grade) {
+        System.out.print(modCode + " has been added to " + semester);
```

Comment:
Magic string. Consider putting the string into a constant variable. If needed, use string formatting to fill in the needed information during runtime. For example: 

```
MOD_ADDED_MSG = "%s has been added to %s. ";
outputMsg = String.format(MOD_ADDED_MSG, modCode, semester);
System.out.println(outputMsg); 
```